node/src/main.rs

```diff
@@ -367,7 +365,7 @@ fn async_main() -> impl Future<Item = (), Error = ()> + Send + 'static {
     let eth_adapter = Arc::new(graph_datasource_ethereum::EthereumAdapter::new(transport));
 
     // Ask Ethereum node for network identifiers
-    info!(
+    debug!(
```

Comment:
Can we keep this file as is? It's useful to see the trying IPFS node message and it's useful to have all connection attempt messages as info. There are more like these and they are all info as well.

Reply:
Works for me! Especially since the starting connection messages only log on startup, so removing them doesn't have any noticeable effect on overall log volume.